pkg/kubectl/cmd/taint.go

```diff
@@ -194,8 +194,6 @@ func (o TaintOptions) validateFlags() error {
 	if !o.all && o.selector == "" {
 		if len(o.resources) < 2 {
 			return fmt.Errorf("at least one resource name must be specified since 'all' parameter is not set")
-		} else {
-			return nil
```

Comment:
:+1: good catch!

This can be simplified:

```go
if !o.all && o.selector == "" && len(o.resources) < 2 {
	return fmt.Errorf("at least one resource name must be specified since 'all' parameter is not set")
}
```

Reply:
while we are here, can you remove the period from the end of the error message two lines above?

```go
return fmt.Errorf("setting 'all' parameter with a non empty selector is prohibited.")
```

Error messages in go should not contain punctuation in the end. :)

Reply:
sorry for delay, I will fix soon.

Reply:
no worries, please take your time. :)